vignettes/shinyMobile.Rmd

```diff
@@ -1,76 +1,140 @@
 ---
 title: "Getting Started"
-author: "David Granjon"
+author: "David Granjon & Veerle van Leemput"
 date: "`r Sys.Date()`"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{Getting Started}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
-
-```{r echo=FALSE}
-knitr::include_url("https://rencontresr2021.rinterface.com/")
-```
-
```

Comment:
@DivadNojnarg I removed your lovely presentation from this vignette (hope you don't mind 😇) - as there are so many changes the code is not up-to-date. I'm going to preserve most of the theoretical content from your slides by putting it in vignettes directly (perhaps also some more deepdive articles)

Reply:
Yes no worry. It is fairly outdated now.